modules/eco-delete-nodemaintenance-cr-web-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+//nodes/nodes/eco-node-maintenance-operator.adoc
+
+:_content-type: PROCEDURE
+[id="eco-resuming-node-maintenance-cr-web-console_{context}"]
+= Resuming a node from maintenance mode in the web console
```

Comment:
I'm getting confused trying to understand why there are multiple modules in the PR that are named either "Resuming a node from maintenance mode in the web console" or "Resuming a node from maintenance mode by using the web console". Is that intentional? In the preview output, I am only seeing the latter version. It looks to me like this is another module file that isn't being included anywhere in the assembly. Please double-check and if necessary, revise the PR to remove the module files that shouldn't be part of the PR.

Reply:
I am confused as well